diff --git a/node_modules/react-scripts/.DS_Store b/node_modules/react-scripts/.DS_Store
new file mode 100644
index 0000000..2436366
Binary files /dev/null and b/node_modules/react-scripts/.DS_Store differ
diff --git a/node_modules/react-scripts/scripts/build.js b/node_modules/react-scripts/scripts/build.js
index 7a3a635..5ec801e 100644
--- a/node_modules/react-scripts/scripts/build.js
+++ b/node_modules/react-scripts/scripts/build.js
@@ -9,8 +9,10 @@
 'use strict';
 
 // Do this as the first thing so that any code reading it knows the right env.
-process.env.BABEL_ENV = 'production';
-process.env.NODE_ENV = 'production';
+// process.env.BABEL_ENV = "production"
+// process.env.NODE_ENV = "production"
+process.env.BABEL_ENV = process.env.NODE_ENV
+process.env.NODE_ENV = process.env.NODE_ENV
 
 // Makes the script crash on unhandled rejections instead of silently
 // ignoring them. In the future, promise rejections that are not handled will
@@ -55,7 +57,8 @@ const argv = process.argv.slice(2);
 const writeStatsJson = argv.indexOf('--stats') !== -1;
 
 // Generate configuration
-const config = configFactory('production');
+// const config = configFactory("production"); 
+const config = configFactory(process.env.NODE_ENV);
 
 // We require that you explicitly set browsers and do not fall back to
 // browserslist defaults.
